showcase.yaml

```diff
@@ -1,219 +1,214 @@
 ---
```

Comment:
I don't want to start another 'quotes-or-not' discussion, this change is rather big because of this, and makes it hard to spot the actual changes you made.

Reply:
I made all of the CF refactoring changes as a separate commit so that they could easily be reviewed by themselves. Sorry, I didn't realize the quotes were a point of contention.